.dockerignore

```diff
@@ -5,4 +5,5 @@ node_modules
 db/*.sqlite3
 *.md
 .git*
+vendor/bundle
```

Comment:
Why is this ignored? Don't we want to keep the vendored gems as part of the image? Or would that persist across deploys/commits?

Reply:
Good Q - The .dockerignore tells COPY and ADD commands to ignore certain files.  In this case, we want to keep the Gems installed inside the image (in vendor/bundle) and NOT overwrite them with the local versions on the workstation.   This allows doing `COPY . /upaya` without trouncing things in the production image.

Reply:
Got it! Thanks for the explanation!